controllers/replicationdestination.go

```diff
@@ -120,6 +120,13 @@ func (r *VolumeSnapshotRestoreReconciler) buildReplicationDestination(replicatio
 		replicationDestination.Spec = replicationDestinationSpec
 	}
 
+	// include custom CA if specified
+	resticCustomCA := resticSecret.Data[ResticCustomCA]
+	if len(resticCustomCA) > 0 {
```

Comment:
Test case here? https://github.com/migtools/volume-snapshot-mover/blob/master/controllers/replicationdestination_test.go

Reply:
The existing tests weren't really set up to check this sort of thing, so I tried adding a field to the test input struct to hold a validation function. I didn't see an easier way to do it.